### DISCUSSION TITLE
vapix API overview for S2212?

### DISCUSSION THREAD

**Discussion by CJP-work-acct** [AxisCommunications/discussions#10] [VAPIX] [answered]
Hi I am new to AXIS development. I have had some luck using VAPIX http calls to the individual cameras but I am not finding as much resources for interrogating the NVR itself. I have the Axis S 2212.

Ideally is there a demo that shows to integrate an app in the style of Axis's Camera Station? Would love to build from their specifically how is the detected artifacts timeline queried by API?

Thank you... I have looked in the repo. I apologize if this is covered somewhere obvious. (That's often the way it is.)

**Comment by mattias-kindborg-at-work** [comment, marked as answer]
Hi @CJP-work-acct!

I don't think we've got a repository for integrating with [AXIS S2212](https://www.axis.com/products/axis-s2212), i.e. you've not missed anything. I'll look around and see if I can find the answer to the question.
